examples/react/basic/src/gen/eliza-ElizaService_connectquery.ts

```diff
@@ -0,0 +1,112 @@
+// Copyright 2021-2023 Buf Technologies, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// @generated by protoc-gen-connect-query v0.4.2 with parameter "target=ts,import_extension="
+// @generated from file eliza.proto (package connectrpc.eliza.v1, syntax proto3)
+/* eslint-disable */
+// @ts-nocheck
+
+import {
+  ConverseRequest,
+  ConverseResponse,
+  IntroduceRequest,
+  IntroduceResponse,
+  SayRequest,
+  SayResponse,
+} from "./eliza_pb";
+import { MethodKind } from "@bufbuild/protobuf";
+import { createQueryService } from "@connectrpc/connect-query";
+
+export const typeName = "connectrpc.eliza.v1.ElizaService";
+
+/**
+ * ElizaService provides a way to talk to Eliza, a port of the DOCTOR script
+ * for Joseph Weizenbaum's original ELIZA program. Created in the mid-1960s at
+ * the MIT Artificial Intelligence Laboratory, ELIZA demonstrates the
+ * superficiality of human-computer communication. DOCTOR simulates a
+ * psychotherapist, and is commonly found as an Easter egg in emacs
+ * distributions.
+ *
+ * @generated from service connectrpc.eliza.v1.ElizaService
+ */
+export const ElizaService = {
+  typeName: "connectrpc.eliza.v1.ElizaService",
+  methods: {
```

Comment:
This is a pretty big change. I assume the export is for `createRouterTransport`?

Reply:
Correct. The plugins already generated this, it just wasn't exported. This just makes it so testing can be more easily made without adding another protoc plugin.

Reply:
Yeah that's fair. I figured it would make sense for the v1 release. The overlap with `protoc-gen-connect-es` isn't ideal. It does feel like every one of our plugins generates the same structure, I wonder if it makes sense to export a utility from `@bufbuild/protoplugin` since `ServiceType` is defined within `@bufbuild/protobuf`?

Reply:
At this point, and considering that the services are relatively simple, we're better off duplicating the code in the plugins. I'm just thinking that it may be a bit confusing from the user perspective, but as I said before, fair for overall simplicity.